solidity/test/bridge/Bridge.test.ts

```diff
@@ -856,6 +853,302 @@ describe("Bridge", () => {
                         }
                       )
 
+                      context(
+                        "when the single input is a revealed unswept deposit with a trusted vault",
+                        async () => {
+                          let vault: FakeContract<IVault>
+                          let tx: ContractTransaction
+
+                          const data: DepositSweepTestData = SingleP2WSHDeposit
+                          // Take wallet public key hash from first deposit. All
+                          // deposits in same sweep batch should have the same value
+                          // of that field.
+                          const { walletPubKeyHash } = data.deposits[0].reveal
+
+                          before(async () => {
+                            await createSnapshot()
+
+                            // Simulate the wallet is an Live one and is known in
```

Comment:
```suggestion
                            // Simulate the wallet is a Live one and is known in
```

Reply:
Addressed in the PR that splits the Bridge tests. See https://github.com/keep-network/tbtc-v2/pull/270/commits/4d57cb9e4fce00b64cc8544cdc18ce22a6d721b1